examples/two_dimensional_turbulence.jl

```diff
@@ -37,6 +37,8 @@ model = NonhydrostaticModel(; grid,
 
 using Statistics
 
+using Random; Random.seed!(1234) #hide
```

Comment:
```suggestion
using Random

Random.seed!(1234)
```

I think we want to show this!